plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala

```diff
@@ -123,9 +127,27 @@ object CassandraOutput {
     val connectionHost = configuration.getString("connectionHost", DefaultHost)
     val connectionPort = configuration.getString("connectionPort", DefaultPort)
 
+    val sparkProperties = getSparkCassandraProperties(configuration)
+
+    sparkProperties ++
     Seq(
       ("spark.cassandra.connection.host", connectionHost),
       ("spark.cassandra.connection.port", connectionPort)
     )
   }
+
+  private def getSparkCassandraProperties(configuration: Map[String, JSerializable]): Seq[(String, String)] = {
+    configuration.get("sparkProperties") match {
+      case Some(properties) =>
+        val conObj = configuration.getMapFromJsoneyString("sparkProperties")
+        conObj.map(c => {
+          val key = c.get("sparkPropertyKey").get
```

Comment:
Please use c("sparktaPropertyKey") if you are assuming that this property is mandatory. Maybe you could use a better name for "c", like propKeyPair for example.